lib/sidekiq/manager.rb

```diff
@@ -26,7 +26,7 @@ class Manager
     def initialize(condvar, options={})
       logger.debug { options.inspect }
       @options = options
-      @count = options[:concurrency] || 25
+      @count = options[:concurrency].to_i || 25
```

Comment:
You probably want to do `(options[:concurrency] || 25).to_i` instead. Because `nil.to_i` is 0 and the 25 default will never be used, making it throw the error `Concurrency of 0 is not supported` if you leave concurrency without any value which is a breaking change